tests/unit/modules/test_role_binding.py

```diff
@@ -0,0 +1,149 @@
+        assert state == 'present'
+        assert path == '/rolebindings/test_role_binding'
+        assert payload == dict(
+            role_ref=dict(
+                name='test_role',
+                type='Role',
+            ),
+            subjects=[
+                dict(
+                    name='test_user',
+                    type='User',
+                ),
+            ],
+            metadata=dict(
+                name='test_role_binding',
+                namespace='default',
+            ),
+        )
+        assert check_mode is False
+
+    def test_minimal_role_binding_parameters_groups(self, mocker):
+        sync_mock = mocker.patch.object(utils, 'sync')
+        sync_mock.return_value = True, {}
+        set_module_args(
+            name='test_role_binding',
+            role='test_role',
+            groups=['test_group'],
+        )
+
+        with pytest.raises(AnsibleExitJson):
+            role_binding.main()
+
+        state, _client, path, payload, check_mode, _compare = sync_mock.call_args[0]
+        assert state == 'present'
+        assert path == '/rolebindings/test_role_binding'
+        assert payload == dict(
+            role_ref=dict(
+                name='test_role',
+                type='Role',
+            ),
+            subjects=[
+                dict(
+                    name='test_group',
+                    type='Group',
+                ),
+            ],
+            metadata=dict(
+                name='test_role_binding',
+                namespace='default',
+            ),
+        )
+        assert check_mode is False
+
+    def test_all_role_binding_parameters(self, mocker):
+        sync_mock = mocker.patch.object(utils, 'sync')
+        sync_mock.return_value = True, {}
+        set_module_args(
+            name='test_role_binding',
+            role='test_role',
+            users=['user_1', 'user_2'],
+            groups=['group_1', 'group_2'],
+        )
+
+        with pytest.raises(AnsibleExitJson):
+            role_binding.main()
+
+        state, _client, path, payload, check_mode, _compare = sync_mock.call_args[0]
+        assert state == 'present'
+        assert path == '/rolebindings/test_role_binding'
+        assert payload == dict(
+            metadata=dict(
+                name='test_role_binding',
+                namespace='default',
+            ),
+            role_ref=dict(
+                name='test_role',
+                type='Role',
+            ),
+            subjects=[
+                dict(
+                    name='group_1',
+                    type='Group',
+                ),
+                dict(
+                    name='group_2',
+                    type='Group',
+                ),
+                dict(
+                    name='user_1',
+                    type='User',
+                ),
+                dict(
+                    name='user_2',
+                    type='User',
+                ),
```

Comment:
Maybe we should compare this the same way we compare subjects in code? Because currently, if the sensu API decides to spice things up and start returning subjects in arbitrary order, our test will fall flat on its face.